sql/database_test.go

```diff
@@ -302,3 +577,6 @@ func TestSchemaDrift(t *testing.T) {
 	require.Regexp(t, `.*\n.*\+.*CREATE TABLE newtbl \(id int\);`,
 		observedLogs.All()[0].ContextMap()["diff"])
 }
+
+// TBD: test WAL modes for temp DB
+// TBD: remove SQLITE_OPEN_WAL from open flags and check journal mode
```

Comment:
Will this be part of this PR or shall we create an issue for it?

Reply:
Removed.
I wrote these comments due to initial misunderstanding of `SQLITE_OPEN_WAL` option. Thing is, per SQLite API docs, this flag is not to be used when opening the database, and is only used for the internal SQLite FS layer. But `crawshaw` alters the use of this flag (which is probably not the best design decision, but that needs to be fixed in crawshaw), which led me to some confusion.
I've added a comment regarding this in the code: https://github.com/spacemeshos/go-spacemesh/blob/26aa21cc2c3fe08d0bb8d0de23f77c4e348240cf/sql/database.go#L250-L255